source/_docs/lockr.md

```diff
@@ -0,0 +1,162 @@
+
+## Why Lockr?
+
+By combining a simple to use developer interface with a managed scalable cloud key management system, Lockr allows applications of all sizes to meet industry standards for key management. Unlike other key managers, Lockr offers a "white glove service" that provides additional layers of security and system monitoring, no ongoing maintenance and continuous development for integrations with your favorite modules and plugins. Offsite API & encryption key management delivers best-practice security to help sites comply with HIPAA, FERPA, and FISMA among others. Development use of Lockr in the Dev, Test and Multi-Dev environments is always free.  
+
+## WordPress Installation
+
+Lockr is automatically configured to secure API keys for numerous 3rd party plugins for seamless integration and securing of your keys. 
+Visit our [Github page](https://github.com/lockr/lockr-patches/tree/wp) for a list of plugins that can be automatically patched.
+
+### Install through the Administration Interface
+
+The official [Lockr Plugin](https://wordpress.org/plugins/lockr) makes it a breeze to get started and quickly secure your keys. Install and activate the latest release through the WordPress dashboard or place it it in /wp-content/plugins directory and activate via the dashboard.
+
+1. Install through the WordPress plugin screen directly or upload the Lockr package to `/wp-content/plugins/` directory. 
+2. Activate Lockr through the “Plugins” section in your WordPress backend  (ex. `http://my-site.pantheonsite.io/wp-admin/plugins.php` )
+3. Visit the Lockr Configuration page  (ex. `http://my-site.pantheonsite.io/wp-admin/admin.php?page=lockr-site-config`)
+
+![WP Lockr Configuration example](/source/docs/assets/images/lockr_wpconfig.png)​
+
+4. Enter your email address and Lockr will take care of the rest! (note: For first time users, you will be receiving an email with instruction, if an existing Lockr users should use the email address they used to create their Lockr account. After submitting the request to register the site, you will be prompted for your Lockr password to confirm adding the site to your account.)
+
+![WP Lockr all keys page example](/source/docs/assets/images/lockr_wpallkeys.png)​
+
+Now you’re set! Your WordPress application is now set up to start entering your keys through the Lockr config. Once you have successfully installed the plugin, you can see a list of keys (and edit/update) by going to **All Keys** under the Lockr Menu. Be sure to visit the [Lockr patch library](https://github.com/lockr/lockr-patches/tree/wp) for the latest patches to your favorite plugins.
+
+
+### WP-CLI Commands
+
+The Lockr plugin contains a number of WP-CLI commands to make registering the site, and setting/getting a key simple through the command line.
+
+```nohighlight
```

Comment:
Edit all commands to use Terminus WP-CLI wrappers - so users don't have to run WP-CLI locally. 

Ex: `terminus wp 'lockr register-site --email=[<Lockr email address>] --password=[<Lockr account password>]'`